project_templates/technote_aastex/{{cookiecutter.series.lower()}}-{{cookiecutter.serial_number}}/abstract.tex

```diff
@@ -3,6 +3,16 @@
 \begin{abstract}
 {%- endraw %}
 {{ cookiecutter.abstract }}
+As the Commissioning Execution Plan (LSE-390) says, "The project team shall
```

Comment:
At the least, please put a new line above this so the user's templated abstract doesn't collide into the same paragraph as this content.